java-cfenv-jdbc/src/main/java/io/pivotal/cfenv/jdbc/SqlServerJdbcUrlCreator.java

```diff
@@ -38,24 +38,38 @@ public class SqlServerJdbcUrlCreator extends AbstractJdbcUrlCreator {
 
 	@Override
 	public String buildJdbcUrlFromUriField(CfCredentials cfCredentials) {
+		String jdbcURL = "";
 		UriInfo uriInfo = cfCredentials.getUriInfo(SQLSERVER_SCHEME);
-		Map<String, String> uriParameters = parseSqlServerUriParameters(uriInfo.getUriString());
-		String databaseName = getDatabaseName(uriParameters);
-		try {
-			URI uri = new URI(uriInfo.getUriString().substring(0, uriInfo.getUriString().indexOf(";")));
-			return String.format("jdbc:%s://%s%s%s%s%s%s",
-					SQLSERVER_SCHEME,
-					uri.getHost(),
-					uri.getPort() > 0 ? ":" + uri.getPort() : "",
-					uri.getPath() != null && !uri.getPath().isEmpty() ? "/" + uri.getPath() : "",
-					databaseName != null ? ";database=" + UriInfo.urlEncode(databaseName) : "",
-					uriParameters.containsKey("user") ? ";user=" +  UriInfo.urlEncode(uriParameters.get("user")) : "",
-					uriParameters.containsKey("password") ? ";password=" +  UriInfo.urlEncode(uriParameters.get("password")) : ""
+		String uriString = uriInfo.getUriString();
+		if (uriString.contains(";")) {
+			Map<String, String> uriParameters = parseSqlServerUriParameters(uriString);
+			String databaseName = getDatabaseName(uriParameters);
+			try {
+				uriString = uriString.substring(0, uriInfo.getUriString().indexOf(";"));
+				URI uri = new URI(uriString);
+				jdbcURL = String.format("jdbc:%s://%s%s%s%s%s%s",
+						SQLSERVER_SCHEME,
+						uri.getHost(),
+						uri.getPort() > 0 ? ":" + uri.getPort() : "",
+						uri.getPath() != null && !uri.getPath().isEmpty() ? "/" + uri.getPath() : "",
+						databaseName != null ? ";database=" + UriInfo.urlEncode(databaseName) : "",
+						uriParameters.containsKey("user") ? ";user=" +  UriInfo.urlEncode(uriParameters.get("user")) : "",
+						uriParameters.containsKey("password") ? ";password=" +  UriInfo.urlEncode(uriParameters.get("password")) : ""
+				);
+			}
+			catch (URISyntaxException e) {
+				throw new RuntimeException(e);
+			}
+		} else {  // deprecated format for SQLServer URL
```

Comment:
```suggestion
		} else {
```

I was thinking about this a little more and we should probably just leave support for both. The change was easy enough. Why rock the boat?